test/plugins/adonis.spec.js

```diff
@@ -0,0 +1,327 @@
+'use strict'
+
+const axios = require('axios')
+const getPort = require('get-port')
+const agent = require('./agent')
+const plugin = require('../../src/plugins/adonis')
+const { setupResolver, Config, Logger } = require('@adonisjs/sink')
+const { ioc } = require('@adonisjs/fold')
+const path = require('path')
+
+const Server = require('@adonisjs/framework/src/Server')
+const Route = require('@adonisjs/framework/src/Route/Manager')
+const RouteStore = require('@adonisjs/framework/src/Route/Store')
+const Request = require('@adonisjs/framework/src/Request')
+const Response = require('@adonisjs/framework/src/Response')
+const Context = require('@adonisjs/framework/src/Context')
+const Exception = require('@adonisjs/framework/src/Exception')
+const BaseExceptionHandler = require('@adonisjs/framework/src/Exception/BaseHandler')
+
+const reqPath = ('../..')
+
+wrapIt()
+
+describe('Plugin', () => {
+  let port
+  let tracer
```

Comment:
Is this used?

Reply:
No, I thought I was going to use it but turned out I never did. Taking it out.